Source/inv.cpp

```diff
@@ -2003,7 +2003,7 @@ int SyncPutItem(int pnum, int x, int y, int idx, WORD icreateinfo, int iseed, in
 	if (!PutItem(pnum, x, y))
 		return -1;
 
-	CanPut(x, y);
+	assert(CanPut(x, y));

```

Comment:
That will omit the call in release builds. Pull the call outside, capture the return value, and assert on that. (Unless you really want to have CanPut only called in debug builds).

Reply:
Yeah, it should only be in debug, this clearly comes from an `assert()` in the original code since it matches the assert in the corresponding local function and `CanPut()` has no side effects.